HLTrigger/JSONMonitoring/plugins/L1TriggerJSONMonitoring.cc

```diff
@@ -34,14 +34,15 @@
 
 struct L1TriggerJSONMonitoringData {
   // variables accumulated event by event in each stream
+  static constexpr const int kPrescaleUndefined = -2;
```

Comment:
please do not duplicate this from the `L1TriggerJSONMonitoring` class

Reply:
maybe you can move the whole block
```c++
  // special values for prescale index checks
  static constexpr const int kPrescaleUndefined = -2;
  static constexpr const int kPrescaleConflict = -1;
```
above the comment on line 36
```
// variables accumulated event by event in each stream
```

and modify `L1TriggerJSONMonitoring` to reuse it from `L1TriggerJSONMonitoringData` ?